x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -212,7 +212,7 @@ describe('CaseActionBar', () => {
       </TestProviders>
     );
 
-    expect(screen.queryByTitle('Change status')).not.toBeInTheDocument();
+    expect(screen.queryByTitle('Change status')).toBeDisabled();
```

Comment:
hmm does this mean that we'll disable the change status button instead of hiding it when a user doesn't have permissions?